qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/utils/DocumentationUtils.java

```diff
@@ -199,6 +200,26 @@ public static MarkupContent getDocumentation(UserTag userTag, boolean markdown)
 
 		return createMarkupContent(documentation, markdown);
 	}
+	
+	public static MarkupContent getDocumentation(CaseOperator operator, boolean markdown) {

```

Comment:
Add Javadoc

Reply:
Call this utility method too for completion at https://github.com/redhat-developer/quarkus-ls/blob/bdafad0b1fdacbe4d450a0cb0b24462ee44a27b2/qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/completions/QuteCompletionsForExpression.java#L522 to have the same renderer in completion and hover .

Reply:
It should be really nice if we could improve the documentation like we have for `or` method value resolver :

![image](https://user-images.githubusercontent.com/1932211/190278694-8ca919e2-1f81-413a-bc8e-977610bd8be1.png)

To do that, you need : 

 * a description
 * a sample (list of String)
 * an URL

In other words Operation should store those information. I suggest that you create a `DocumentableItem` interface like this:

```java
package com.redhat.qute.parser.template;

import java.util.List;

public interface DocumentableItem {

	String getDescription();
	
	List<String> getSample();
	
	String getUrl();
}

```

This interface should be implemented with:

 * Operator
 * MethodValueResolver

After that you can extract this code 
https://github.com/redhat-developer/quarkus-ls/blob/bdafad0b1fdacbe4d450a0cb0b24462ee44a27b2/qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/utils/DocumentationUtils.java#L77 in a new method which have DocumentableItem as parameter